pkg/scheduler/framework/interface.go

```diff
@@ -47,12 +47,24 @@ type NodeScore struct {
 	Score int64
 }
 
-// PluginToNodeScores declares a map from plugin name to its NodeScoreList.
-type PluginToNodeScores map[string]NodeScoreList
-
 // NodeToStatusMap declares map from node name to its status.
 type NodeToStatusMap map[string]*Status
 
+// NodePluginScores is a struct with node name and scores for that node.
+type NodePluginScores struct {
+	// Name is node name.
+	Name string
+	// Scores is scores from plugins and extenders.
+	Scores []SchedulingScore
+}
+
+// SchedulingScore is a struct with plugin/extender name and score.
+type SchedulingScore struct {
+	// Name is the name of plugin or extender.
```

Comment:
In this PR, this `Name` field is only used for plugin name.
But it will be used for extender name in #108390. (`[]framework.NodePluginScores` will be used to return all scoring results from extenders and plugins in `prioritizeNodes`. )